topics/zeit/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: ZEIT
+github_url: https://github.com/zeit
+logo: zeit.png
+short_description: Our mission is to make cloud computing as easy and accessible as mobile computing.
+topic: zeit
+url: https://zeit.co
+---
+Our mission is to make cloud computing as easy and accessible as mobile computing.
```

Comment:
Ditto prev comment; a couple sentences about what Zeit is, using a more neutral POV would be better!

Reply:
@nayafia ✅